File: trail2/두 개의 동일한 수열/two-equal-series.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int n;

bool isSameSeq(int *arr_A, int *arr_B) {
    for (int i = 0; i < n; i++) {
        if (arr_A[i] != arr_B[i]) {
            return false;
        }
    }
    return true;
}

int main() {
    // Please write your code here.
    cin >> n;

    int arr_A[n], arr_B[n];
    for (int i = 0; i < n; i++) {
        cin >> arr_A[i];
    }
    for (int i = 0; i < n; i++) {
        cin >> arr_B[i];
    }

    sort(arr_A, arr_A + n);
    sort(arr_B, arr_B + n);

    if (isSameSeq(arr_A, arr_B))
        cout << "Yes" << endl;
    else
        cout << "No" << endl;

    
    return 0;
}

File: trail2/두 개의 동일한 수열/two-equal-series.py
n = int(input())
a = list(map(int, input().split()))
b = list(map(int, input().split()))

def is_same():
    for i in range(n):
        if (a[i] != b[i]):
            return False
    return True

a.sort()
b.sort()
if(is_same()):
    print("Yes")
else:
    print("No")